keyboards/keebio/quefrency/keymaps/kingwangwong/keymap.c

```diff
@@ -0,0 +1,40 @@
+enum layer_names {
+    _BASE,
+    _FN1
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
```

Comment:
Doesn't look like you're using this keycode anymore, so you can go ahead and remove the enum.